x/denommetadata/types/errors.go

```diff
@@ -8,5 +8,7 @@ import (
 
 // x/denommetadata module sentinel errors
 var (
-	ErrUnknownRequest = errorsmod.Register(ModuleName, 1002, "unknown request")
+	ErrDenomAlreadyExists = errorsmod.Register(ModuleName, 1000, "denom metadata is already registered")
+	ErrUnknownRequest     = errorsmod.Register(ModuleName, 1002, "unknown request")
+	ErrRollappNotFound    = errorsmod.Register(ModuleName, 1003, "rollapp not found")
 )
```

Comment:
we should use gerrc as a basis of errors now

https://github.com/dymensionxyz/gerr-cosmos/blob/main/gerrc/errors.go#L26-L41